homeassistant/components/unifi/device_tracker.py

```diff
@@ -54,10 +62,8 @@
 })
 
 
-def get_scanner(hass, config):
+async def async_get_scanner(hass, config):
```

Comment:
If you're going to make it part of the config flow, all this is going away and you need to inherit from the device tracker entity instead. So I am not sure if this conversion makes sense if it is done because of the transition to a config flow.

Reply:
Im doing it in two steps to make sure it doesn't break anything